src/client/components/equalizer/equalizer.js

```diff
@@ -110,19 +115,22 @@ class Equalizer extends Component {
 
   renderEqualizer = () => {
     let {analyser, heightArray, ctx, numPoints} = this.state;
-    let flagColor = true;
+    let isFirstColorForEqualizerUsed = true;
     analyser.getByteFrequencyData(heightArray);
-    let width = ctx.canvas.width;
-    let height = ctx.canvas.height;
-    let countcolumns = Math.floor(ctx.canvas.width / 52);
-    let columnwidth = Math.floor(5 / 6 * ctx.canvas.width / 52);
+    let {width, height} = ctx.canvas;   
+    let numbersOfRectengle = 52;
+    let totalAreaOfRectangles = 5/6;
+    let rectangleCornerRadius = 2;
+    let rectangleMaxWidth = 512;
+    let countColumns = Math.floor(ctx.canvas.width / numbersOfRectengle);
```

Comment:
you already have this ctx.canvas.width in width variable, why not use here?

Reply:
ok